tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -605,6 +605,23 @@ public function testWhereBetweens()
         $this->assertEquals([], $builder->getBindings());
     }
 
+    public function testWhereBetweenColumns()
```

Comment:
`testWhereBetweenColumn()`

Reply:
all other are in plural form, `testWhereBetweens()´ for example.